common/supplemental/supplementalData.xml

```diff
@@ -2784,6 +2784,7 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="zh" populationPercent="90" officialStatus="official"/>	<!--Chinese-->
 			<languagePopulation type="wuu" populationPercent="6"/>	<!--Wu Chinese-->
 			<languagePopulation type="yue_Hans" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Simplified)-->
+			<languagePopulation type="yue_Hant" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Traditional)-->
```

Comment:
Since Hant is the default for yue, this should just be:
```suggestion
			<languagePopulation type="yue" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Traditional)-->
```

Reply:
Okay, but we're not changing the default in this ticket since we're still waiting for input, so I'll change yue_Hans to yue for now.